monai/transforms/utils.py

```diff
@@ -690,3 +691,42 @@ def extreme_points_to_image(
     points_image = (points_image - min_intensity) / (max_intensity - min_intensity)
     points_image = points_image * (rescale_max - rescale_min) + rescale_min
     return points_image
+
+
+def map_spatial_axes(
+    img: np.ndarray,
```

Comment:
this could be `img_ndim: int`

Reply:
Sure, update in the latest comment.
Thanks.